.github/ISSUE_TEMPLATE/plugin_question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Plugin related question
+about: I have a question whether I can use certain plugins in flutter-desktop
+title: ""
+labels: ""
+assignees: ""
+---
+
+<!--
+You can find list of available plugins [here](https://github.com/google/flutter-desktop-embedding/tree/master/plugins)
```

Comment:
This isn't necessarily true; hopefully macOS plugin implementations will start being available in other places before long. (And theoretically people could host unpublished Windows and Linux plugins as FDE currently does, they would just be subject to breakage so I wouldn't necessary recommend it.)